File: src/test/java/net/slipp/domain/qna/AnswerTest.java
package net.slipp.domain.qna;

import static net.slipp.domain.qna.AnswerBuilder.*;

import static org.hamcrest.CoreMatchers.*;
import static org.junit.Assert.*;

import org.junit.Test;

public class AnswerTest {
	@Test
	public void likedMoreThan() {
		Answer answer = anAnswer().withTotalLiked(2).build();
		assertThat(answer.likedMoreThan(2), is(true));
	}

	@Test
	public void isNotBest() throws Exception {
		Answer answer = anAnswer().withTotalLiked(1).build();
		assertThat(answer.likedMoreThan(2), is(false));		
	}
}

File: src/test/scala/net/slipp/domain/qna/AnswerTest.scala
package net.slipp.domain.qna

import net.slipp.domain.qna.AnswerBuilder._
import net.slipp.support.test.UnitTest
import org.hamcrest.CoreMatchers._
import org.junit.Assert._
import org.junit.Test

class AnswerTest extends UnitTest {
  @Test def likedMoreThan {
    val answer = aSomeAnswer(sumLiked = 2)
    assertThat(answer.likedMoreThan(2), is(true))
  }

  @Test def isNotBest {
    val answer = aSomeAnswer(sumLiked = 1)
    assertThat(answer.likedMoreThan(2), is(false))
  }
}